cdap-ui/app/features/adapters/controllers/tabs/runs/tabs/status-ctrl.js

```diff
@@ -4,15 +4,18 @@ angular.module(PKG.name + '.feature.adapters')
 
     $scope.transforms = [{
       name: '',
-      properties: {}
+      properties: {},
```

Comment:
Can you add a TODO comment on top of this controller that we should use ```rAdapterDetail``` resolved variable. Right now we are making two different calls to backend for the same data. 
It could be something like this,
```
//TODO: We should use rAdapterDetail here since this data is already resolved at adapter.detail state
```